### PR TITLE
<v2.9> Converted test_wl_pause_orchestration from python to go

### PR DESCRIPTION
**Pull request by lscalabrini01**

### CODE REVIEW

**Review by floatingman — DISMISSED** [review]
LGTM

File: tests/v2/actions/workloads/deployment/deployment.go
Comment (slickwarren) [inline tests/v2/actions/workloads/deployment/deployment.go:1]: updateDeployment wasn't ported here, should it be?

**Review by slickwarren — CHANGES_REQUESTED** [review]
I don't think this has been rebased from your latest changes on main, right?

**Comment by lscalabrini01** [comment]
> I don't think this has been rebased from your latest changes on main, right?

File: tests/v2/actions/workloads/deployment/deployment.go
Comment (lscalabrini01) [reply tests/v2/actions/workloads/deployment/deployment.go:1]: Yes, it hasn't been rebased from my latest changes on main. I'm waiting for the main changes to be merged before updating the backports

**Review by slickwarren — APPROVED** [review]

**Review by floatingman — DISMISSED** [review]
LGTM

**Review by slickwarren — APPROVED** [review]

**Review by anupama2501 — APPROVED** [review]

**Pull request merged**